BUILD.md

```diff
@@ -283,7 +283,7 @@ cmake --build build
 General requirements (see [#1469](https://github.com/neovim/neovim/issues/1469#issuecomment-63058312)):
 
 - Clang or GCC version 4.9+
-- CMake version 3.13+, built with TLS/SSL support
+- CMake version 3.16+, built with TLS/SSL support
   - Optional: Get the latest CMake from an [installer](https://github.com/Kitware/CMake/releases) or the [Python package](https://pypi.org/project/cmake/) (`pip install cmake`)
```

Comment:
maybe https://cmake.org/download/ is a better link. it also mentions pip, snap, and ubuntu apt repo.

Reply:
Added to https://github.com/neovim/neovim/pull/30177